src/main/webapp/starbucks-map.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Starbucks</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCnvi8htmNmKQkmIMqiCu1KGgT3ikR6uO4"></script>
+  <script>  
+    function createStarbucksMap(){
+      fetch('/starbucks-map').then(function(response) {
+        return response.json();
+      }).then((Starbucks) => {
+        const map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 40.7128, lng: -74.0060},
+          zoom: 14
+        });
+        Starbucks.forEach((Starbucks) => {
```

Comment:
Variable names should start with a lowercase letter, and we want to make sure the reader distinguishes between the collection of stores and each individual store, so let's make this:

```
starbucksArray.forEach((starbucks) =>
```